pybricks/common/pb_type_speaker.c

```diff
@@ -293,7 +333,7 @@ STATIC mp_obj_t pb_type_Speaker_play_notes(size_t n_args, const mp_obj_t *pos_ar
     mp_obj_t iterable = mp_getiter(notes_in, NULL);
     if (nlr_push(&nlr) == 0) {
         while ((item = mp_iternext(iterable)) != MP_OBJ_STOP_ITERATION) {
-            pb_type_Speaker_play_note(self, item, duration);
+            pb_type_Speaker_play_note(self, item, duration, self->sample_attenuator);
```

Comment:
We already pass `self` as an argument, so shouldn't need to pass `self->sample_attenuator` separately.

Reply:
Very good catch, thanks! Fixed.